tests/ui/codegen/target-cpus.rs

```diff
@@ -1,3 +1,9 @@
 //@ needs-llvm-components: webassembly
 //@ compile-flags: --print=target-cpus --target=wasm32-unknown-unknown
 //@ check-pass
+
+// LLVM at HEAD has added support for the `lime1` CPU. Remove it from the
+// output so that the stdout with LLVM-at-HEAD matches the output of the LLVM
+// versions currently used by default.
+// FIXME: Once Rust upgrades to LLVM 20, remove this.
+//@ normalize-stdout-test: "(?m)^ *lime1\n" -> ""
```

Comment:
Suggestion: can u open an issue to track this FIXME so we don't forget?

Reply:
Done: https://github.com/rust-lang/rust/issues/133919